01-front-end-basics/jquery-dom-manipulation/24/README.md

```diff
@@ -0,0 +1,252 @@
+
+### DOM Analogy: The Human Body
+
+* HTML = Skeleton
+* Javascript = Muscles, Brain, and Organs
+* CSS = Skin and Clothing
+
+![muscles](https://cloud.githubusercontent.com/assets/7833470/10700126/8e2997da-7970-11e5-96ba-1e5eaf793394.jpg)
+
+## Intro to jQuery
+
+jQuery is a **powerful, easy-to-use JavaScript library**. It was invented in 2005 and has made our lives as developers better ever since. The slogan for jQuery is "write less, do more". jQuery is a great toolbox that provides convenient functions we can use as opposed to writing the functionality we need from scratch.
+
+*We use jQuery because it's:*
+
+* **Convenient:** solves problems developers commonly face.
+* **Less Buggy:** ensures JavaScript DOM manipulation works the same, cross-browser.
+* **Modern:** brings JavaScript DOM manipulation into the 21st century.
+* **Popular:** 26.95% of all sites use jQuery!
+
+### jQuery vs. Vanilla JS
+
+It's important to note that any DOM manipulation done with jQuery can also be done with pure (vanilla) JavaScript. Some argue <a href="http://youmightnotneedjquery.com" target="_blank">you might not need jQuery</a>. It can be overkill, depending on what you're doing.
+
+Take a look at this <a href="http://bl.ocks.org/joyrexus/5322252" target="blank">Comparison of jQuery and Vanilla JS</a>. Here are some of the basic differences:
+
+**Selecting Elements**
+
+```js
+// jquery
+var $divs = $('div');
+
+// vanilla js
+var divs = document.querySelectorAll('div');
+```
+
+**Selecting Elements by Class**
+
+```js
+// jquery
+var $content = $('.content');
+
+// vanilla js
+var content = document.getElementsByClassName('content');
+```
+
+**Selecting Elements by Id**
+
+```js
+// jquery
+var $about = $('#about');
+
+// vanilla js
+var about = document.getElementById('about');
+```
+
+**Creating Elements**
+
+```js
+// jquery
+var $newDiv = $('<div></div>');
+$('body').append($newDiv);
+
+// vanilla js
+var newDiv = document.createElement('div');
+document.body.appendChild(newDiv);
+```
+
+### Including jQuery
+
+To use jQuery, you must include the library in your page. The quickest way to include jQuery in your project is to copy the link to the CDN (content delivery network) and put it into a script tag at the bottom of your HTML `<body>`. You can get the CDN by searching for "jQuery" on <a href="https://cdnjs.com" target="_blank">cdnjs</a>.
+
+**Important:** Your JavaScript file MUST come after jQuery in the order you require scripts. This is because the page loads in order, so the jQuery library must be loaded before you can use any of it's functionality.
```

Comment:
Ah apparently I've been reviewing 24's lesson.  Should I do that ?  at least for typos?